### PR TITLE
ROCANA-7884: Create a jenkins job for testing metrics-jvm-nonaccumulating

### PR DESCRIPTION
**Pull request by joey**
- Add the maven repos to deploy artifacts to

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -7,6 +7,9 @@ plugins {
 repositories {
     mavenLocal()
     mavenCentral()
+    maven {
+        url "http://repository.rocana.com/content/groups/com.scalingdata"
```
Comment (basharaj) [inline build.gradle:11]: shouldn't we publish this to `rom.rocana`?

File: build.gradle [hunk 2]
```diff
@@ -40,3 +43,17 @@ task wrapper(type: Wrapper) {
     group = "build"
     gradleVersion = '2.4'
 }
+
+uploadArchives {
+    repositories {
+        mavenDeployer {
+            repository(url: "http://repository.rocana.com/content/repositories/com.scalingdata.releases") {
+                authentication(userName: rocanaReleaseUser, password: rocanaReleasePassword)
```
Comment (basharaj) [inline build.gradle:52]: Same here

**Review by basharaj — APPROVED** [review]

**Comment by actgardner** [comment]
LGTM

File: build.gradle (hunk 1 above)
Comment (joey) [reply build.gradle:11]: We don't have a `com.rocana` in our nexus, so artifacts are still published here.

**Pull request merged**